src/engines/html5/html5.js

```diff
@@ -39,6 +39,18 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @private
    */
   _mediaSourceAdapter: ?IMediaSourceAdapter;
+  /**
+   * The last time detach occurred
+   * @type {number}
+   * @private
+   */
+  _lastTimeDetach: number = 0;
+  /**
+   * The source object for reattaching media.
```

Comment:
`for reattaching media.` is redundant for description
```suggestion
   * The source object
```